next-tavla/app/(admin)/boards/utils/updateTags.ts

```diff
@@ -0,0 +1,33 @@
+import { userCanWriteBoard } from 'Admin/utils/firebase'
+import { getUserFromSessionCookie } from 'Admin/utils/formActions'
+import { firestore } from 'firebase-admin'
+import { uniq } from 'lodash'
+import { redirect } from 'next/navigation'
+import { TTag } from 'types/meta'
+import { TBoardID } from 'types/settings'
+
+async function fetchTags({ bid }: { bid: TBoardID }) {
+    const board = await firestore().collection('boards').doc(bid).get()
+    if (!board.exists) throw 'board/not-found'
+    const user = await getUserFromSessionCookie()
+    if (!user) redirect('/')
```

Comment:
Denne redirecten vil ikke fungere siden du kaller fetchTags inni en trycatch blokk. Returner `operation-not-allowed` istedenfor